machine-check/examples/esm/example.test.ts

```diff
@@ -49,7 +49,7 @@ const subscriptions = {
 
 describe('swarmProtocol', () => {
   it('should be well-formed', () => {
-    expect(checkSwarmProtocol(swarmProtocol, subscriptions)).toEqual({ type: 'OK' })
+    expect(checkSwarmProtocol(swarmProtocol, subscriptions)).not.toEqual({ type: 'OK' })
```

Comment:
I’d rather check that the errors are as expected and/or fixing the protocol to make it well-formed.

Reply:
Should we create cases for when the SwarmProtocol is well-formed?

Reply:
yes, thanks!